src/pages/home/report/ContextMenu/ContextMenuActions.js

```diff
@@ -130,6 +133,7 @@ export default [
         getDescription: () => {},
     },
     {
+        isAnonymousAction: false,
```

Comment:
In [this issue](https://github.com/Expensify/App/issues/27039), we decided to allow anonymous user to copy url and email.